xml/ha_storage_protection.xml

```diff
@@ -1373,6 +1373,429 @@ Illegal request, Invalid opcode</screen>
+    </step>
+    <step>
+      <para>
+        Adjust the other settings as required. For more information,
+        see <xref linkend="pro-ha-storage-protect-sbd-config"/>.
+      </para>
+    </step>
+    <step>
+      <para>
+        Copy the configuration file to all nodes:
+      </para>
+<screen>&prompt.root;<command>csync2 -xv</command></screen>
+    </step>
+    <step>
+      <para>
+        Start the cluster services on all nodes:
+      </para>
+<screen>&prompt.root;<command>crm cluster start --all</command></screen>
+    </step>
+    <step>
+      <para>
+        Check the status of the cluster:
+      </para>
+<screen>&prompt.root;<command>crm status</command></screen>
+      <para>
+        Initially, the nodes have a status of <literal>UNCLEAN (offline)</literal>,
+        but after a short time they change to <literal>Online</literal>.
+      </para>
+    </step>
+    <step>
+      <para>
+        Check the SBD configuration. First, check the device metadata:
+      </para>
+<screen>&prompt.root;<command>sbd -d /dev/disk/by-id/<replaceable>DEVICE_ID</replaceable> dump</command>
+</screen>
+      <para>
+        Then check that all nodes in the cluster are assigned to a slot in the device:
+      </para>
+<screen>&prompt.root;<command>sbd -d /dev/disk/by-id/<replaceable>DEVICE_ID</replaceable> list</command></screen>
+    </step>
+    <step>
+      <para>
+        Check the status of the SBD service:
+      </para>
+<screen>&prompt.root;<command>systemctl status sbd</command></screen>
+      <para>
+        If you changed diskless SBD to disk-based SBD, check that the following section
+        includes a device ID:
+      </para>
+<screen>CGroup: /system/.slice/sbd.service
+        |&mdash;23314 "sbd: inquisitor"
+        |&mdash;23315 "sbd: watcher: /dev/disk/by-id/<replaceable>DEVICE_ID</replaceable> - slot: 0 - uuid: <replaceable>DEVICE_UUID</replaceable>"
+        |&mdash;23316 "sbd: watcher: Pacemaker"
+        |&mdash;23317 "sbd: watcher: Cluster"</screen>
+    </step>
+    <step>
+      <para>
+        If you changed any timeouts, or if you changed diskless SBD to disk-based SBD,
+        you might also need to change the CIB properties <literal>stonith-timeout</literal> and
+        <literal>stonith-watchdog-timeout</literal>. For disk-based SBD,
+        <literal>stonith-watchdog-timeout</literal> should be <literal>0</literal>.
+        For more information, see <xref linkend="sec-ha-storage-protect-watchdog-timings"/>.
+      </para>
+      <para>
+        To check the current values, run the following command:
+      </para>
+<screen>&prompt.root;<command>crm configure show</command></screen>
+      <para>
+        If you need to change the values, use the following commands:
+      </para>
+<screen>&prompt.root;<command>crm configure property stonith-watchdog-timeout=0</command>
+&prompt.root;<command>crm configure property stonith-timeout=<replaceable>VALUE</replaceable></command></screen>
+    </step>
```

Comment:
It's occurring to me that for the case of changing diskless SBD to disk-based SBD, we still need to "Configure the SBD STONITH resource" as how it's done in this procedure:

https://susedoc.github.io/doc-sleha/main/single-html/SLE-HA-administration/#pro-ha-storage-protect-fencing

Reply:
Uh oh, you're right, thanks for catching that! To keep it simple, I'll just add a step linking to this procedure.